### ISSUE TITLE
Power Apps CLI Output Encoding

### ISSUE THREAD

**Issue by CoreySutton** [microsoft/powerplatform-build-tools#589]
**Context**
Running the Power Apps CLI (PAC) via PowerShell task in Azure DevOps build pipeline. The PowerShell task relies on parsing the output of each PAC command that runs.

**Issue**
The output of PAC contains Unicode character �.

Expected output
```bash
2023-10-17T12:08:24.2992625Z Microsoft PowerPlatform CLI
2023-10-17T12:08:24.2994062Z Version: 1.27.6+g9f62afb
2023-10-17T12:08:24.2995091Z Online documentation: https://aka.ms/PowerPlatformCLI
2023-10-17T12:08:24.2996580Z Feedback, Suggestions, Issues: https://github.com/microsoft/powerplatform-build-tools/discussions
2023-10-17T12:08:24.2997462Z 
2023-10-17T12:08:24.2998368Z Error: No profiles were found on this computer. Please run 'pac auth create' to create one.
2023-10-17T12:08:24.2999206Z 
2023-10-17T12:08:24.2999754Z Usage: pac auth select [--index] [--name]
2023-10-17T12:08:24.3000239Z 
2023-10-17T12:08:24.3001843Z   --index                     The index of the profile to be active. (alias: -i)
2023-10-17T12:08:24.3002979Z   --name                      The name of the profile to be active (alias: -n)
```

Current output
```bash
2023-10-18T02:11:57.5923799Z M�i�c�r�o�s�o�f�t� �P�o�w�e�r�P�l�a�t�f�o�r�m� �C�L�I�
2023-10-18T02:11:57.5925256Z �
2023-10-18T02:11:57.5925778Z �V�e�r�s�i�o�n�:� �1�.�2�8�.�2�+�g�d�0�0�3�e�f�3�
2023-10-18T02:11:57.5926348Z �
2023-10-18T02:11:57.5927051Z �O�n�l�i�n�e� �d�o�c�u�m�e�n�t�a�t�i�o�n�:� �h�t�t�p�s�:�/�/�a�k�a�.�m�s�/�P�o�w�e�r�P�l�a�t�f�o�r�m�C�L�I�
2023-10-18T02:11:57.5927821Z �
2023-10-18T02:11:57.5928992Z �F�e�e�d�b�a�c�k�,� �S�u�g�g�e�s�t�i�o�n�s�,� �I�s�s�u�e�s�:� �h�t�t�p�s�:�/�/�g�i�t�h�u�b�.�c�o�m�/�m�i�c�r�o�s�o�f�t�/�p�o�w�e�r�p�l�a�t�f�o�r�m�-�b�u�i�l�d�-�t�o�o�l�s�/�d�i�s�c�u�s�s�i�o�n�s�
2023-10-18T02:11:57.5930165Z �
2023-10-18T02:11:57.5930472Z �
2023-10-18T02:11:57.5930777Z �
2023-10-18T02:11:57.5931810Z �E�r�r�o�r�:� �N�o� �p�r�o�f�i�l�e�s� �w�e�r�e� �f�o�u�n�d� �o�n� �t�h�i�s� �c�o�m�p�u�t�e�r�.� �P�l�e�a�s�e� �r�u�n� �'�p�a�c� �a�u�t�h� �c�r�e�a�t�e�'� �t�o� �c�r�e�a�t�e� �o�n�e�.�
2023-10-18T02:11:57.5932877Z �
2023-10-18T02:11:57.5933178Z �
2023-10-18T02:11:57.5933481Z �
2023-10-18T02:11:57.8336770Z �
```

**Comment by petrochuk** [comment]
Can you provide a little bit more details? Which operating system is that? Version of Power Shell etc.

**Comment by Jon730** [comment]
In https://www.nuget.org/packages/Microsoft.PowerApps.CLI
Release Notes for 1.28.2 has:
Misc
 - PAC CLI is outputting in Unicode (UTF16) by default

**Comment by tehcrashxor** [comment]
The encoding changes in 1.28.2 were working well for us when we tested via cmd, PowerShell, and PowerShell Core on Windows, zhs and bash on Mac, and bash on Ubuntu via either WSL or CodeSpaces, but looks like ADO pipelines were breaking on any terminal in ADO.

Hotfix released as v1.28.3 with, reducing the Unicode changes in scope from everything down to just the new `pac power-fx` commands.

**Comment by CoreySutton** [comment]
Thanks for your responses @jon730 and @tehcrashxor. I was able to work around the encoding issue in Azure DevOps Pipelines by forcing an old version. 

Great to hear about the hotfix, will revert to using the latest version.
